libpod/networking_linux.go

```diff
@@ -973,6 +973,13 @@ func (c *Container) getContainerNetworkInfo() (*define.InspectNetworkSettings, e
 		return nil, err
 	}
 
+	// If this a container with a netns and on the default network
+	if c.state.NetNS != nil && isDefault {
```

Comment:
I don't think this is correct... Docker does not populate Networks when only the default is in use, last I checked? And, AFAIK, the original request was not about Networks, it was about populating the `Bridge` field in `InspectNetworkSettings`?

Reply:
Hm. Nevermind, Docker is populating additional networks, even when only the default is specified. I stand corrected.

Reply:
docker does populate it, i checked.  also docker does not populate the bridge field apparently?
```sudo docker inspect foo | jq .[0].NetworkSettings.Bridge
""```